File: app/src/main/java/com/joshrap/liteweight/models/user/FriendRequest.java
package com.joshrap.liteweight.models.user;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class FriendRequest {

    private String userId;
    private String username;
    private String profilePicture;
    private boolean seen;
    private String sentUtc;

}

File: api/LiteWeightApi/Domain/Users/FriendRequest.cs
using Google.Cloud.Firestore;
using LiteWeightAPI.Domain.Converters;
using NodaTime;

namespace LiteWeightAPI.Domain.Users;

[FirestoreData]
public class FriendRequest
{
	[FirestoreProperty("userId")]
	public required string UserId { get; set; }

	[FirestoreProperty("username")]
	public required string Username { get; set; }

	[FirestoreProperty("profilePicture")]
	public required string ProfilePicture { get; set; }

	[FirestoreProperty("seen")]
	public bool Seen { get; set; }

	[FirestoreProperty("sentUtc", ConverterType = typeof(InstantConverter))]
	public Instant SentUtc { get; set; }
}